<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEquipos</class>
 <widget class="QDialog" name="DialogEquipos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestión de Equipos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(113, 177, 129);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgb(1, 107, 97);
    color: white;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 100px;
}
QPushButton:hover {
    background-color: rgb(113, 177, 129);
}</string>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Close</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTitulo">
     <property name="font">
      <font>
       <pointsize>32</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>GESTIÓN DE EQUIPOS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameContenido">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(113, 177, 129);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_main">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="lblListaEquipos">
          <property name="sizeIncrement">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"> background-color: rgb(1, 107, 97);
    color: white;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 100px;</string>
          </property>
          <property name="text">
           <string>Lista de Equipos</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableEquipos">
          <property name="styleSheet">
           <string notr="true">QTableWidget {
    background-color: #93da97;
    border: none;
    border-radius: 5px;
}
QTableWidget::item:selected {
    background-color: #93da97;
    color: rgb(1, 107, 97);
}
QHeaderView::section {
	
	background-color: rgb(1, 107, 97);
    color: white;
    padding: 8px;
    border: 2px solid #93da97 ;;
    font-weight: bold;
}

</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nombre</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Curso</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Color</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Emblema</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnNuevoEquipo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Crear un nuevo equipo</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(1, 107, 97);
    color: white;
    border-radius: 8px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #93da97;
    color: rgb(1, 107, 97);
}
QPushButton:pressed {
    background-color: rgb(1, 107, 97);
}

</string>
            </property>
            <property name="text">
             <string>Nuevo Equipo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditarEquipo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Editar equipo seleccionado</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
 
	background-color: rgb(1, 107, 97);
    color: white;
   
    padding: 8px;
}
QPushButton:hover {
    background-color: #93da97;
    color: rgb(1, 107, 97);
}
QPushButton:pressed {
    background-color: rgb(1, 107, 97);
}</string>
            </property>
            <property name="text">
             <string>Editar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEliminarEquipo">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Eliminar equipo seleccionado</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
 background-color: rgb(1, 107, 97);
    color: white;
      padding: 8px;
}
QPushButton:hover {
    background-color: #93da97;
    color: rgb(1, 107, 97);
}
QPushButton:pressed {
	background-color: rgb(1, 107, 97);
}</string>
            </property>
            <property name="text">
             <string>Eliminar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="lblJugadoresEquipo">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
 background-color: rgb(1, 107, 97);
    color: white;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 100px;</string>
          </property>
          <property name="text">
           <string>Jugadores del Equipo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblEquipoSeleccionado">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
padding: 4px;
 background-color: rgb(1, 107, 97);
    color: white;
    border-radius: 8px;
    padding: 8px 20px;
    font-weight: bold;
    min-width: 100px;</string>
          </property>
          <property name="text">
           <string>Selecciona un equipo para ver sus jugadores</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listJugadores">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(147, 218, 151);</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnAgregarJugador">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Agregar jugador al equipo seleccionado</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(1, 107, 97);</string>
            </property>
            <property name="text">
             <string>Agregar Jugador</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnQuitarJugador">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="toolTip">
             <string>Quitar jugador del equipo</string>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(1, 107, 97);</string>
            </property>
            <property name="text">
             <string>Quitar Jugador</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEquipos</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
